lib/cgo/cipher.hash.go

```diff
@@ -59,14 +61,6 @@ func SKY_cipher_SHA256_Xor(_g *C.cipher__SHA256, _b *C.cipher__SHA256, _arg1 *C.
 	return
 }
 
-//export SKY_cipher_SumSHA256
-func SKY_cipher_SumSHA256(_b []byte, _arg1 *C.cipher__SHA256) (____error_code uint32) {
```

Comment:
why was this removed ?

Reply:
Because to move https://github.com/simelo/libskycoin/blob/stdevAlDen_t7_refactor_in_tests/lib/cgo/cipher.crypto.hw.go#L75